internal/command/commands/service.go

```diff
@@ -302,9 +297,8 @@ func (s *Service) Help() string {
 	if s.Create {
 		return commandHelper("create", "service", []string{
 			"--file=yaml file to read service definition",
-			"--rebuild=rebuild existing service",
-			"--name=name of the service (required if using --rebuild)",
-			"--version=version of the service (required if using --rebuild)",
+			"--name=name of the service",
```

Comment:
After removing the rebuild flag, in which case will we use --name and --version params without the --file param?